packages/application/src/index.ts

```diff
@@ -39,7 +39,7 @@ export { type IPlugin };
  * UI applications with the ability to be safely extended by third
  * party code via plugins.
  */
-export class Application<T extends Widget = Widget> {
+export class Application<T extends Widget | HTMLElement = Widget> {
```

Comment:
Maybe it could be useful to detail the benefit for providing an `HTMLElement` directly, instead of creating a `new Widget()` with its `node` option?

Reply:
Thanks!